packages/react/src/components/UIShell/README.md

```diff
@@ -1,5 +1,77 @@
 # @carbon-labs/react-ui-shell
 
+![npm version](https://img.shields.io/npm/v/@carbon-labs/react-ui-shell)
+![License](https://img.shields.io/github/license/carbon-design-system/carbon-labs)
+
+The `@carbon-labs/react-ui-shell` package extends UI Shell components from
+`@carbon/react`, providing additional enhancements while maintaining
+compatibility.
+
+## 📦 Getting started
+
+To install `@carbon-labs/react-ui-shell` in your project, you will need to run
+the following command using [npm](https://www.npmjs.com/):
+
+```bash
+npm install -S @carbon-labs/react-ui-shell @carbon/react
+```
+
+If you prefer [Yarn](https://yarnpkg.com/en/), use the following command
+instead:
+
+```bash
+yarn add @carbon-labs/react-ui-shell @carbon/react
+```
+
+## ⚡ Usage
+
+To use this package you will need to import components from both `@carbon/react`
+and `@carbon-labs/react-ui-shell` to compose the pieces of the shell together:
```

Comment:
Maybe we can add a note or a list of which components need to be imported from this package instead of `@carbon/react`, just to be clear

Reply:
I was on the fence about how much details to include in the readme, but I can update to add the list of components.